### PR TITLE
Collections and exp2collections (RFC)

### PR DESCRIPTION
**Pull request by pcm32**
This PR adds support for collections on SCXA. This still needs testing on db-scxa module.

### CODE REVIEW

**Comment by pcm32** [comment]
@alfonsomunozpomer let me know of any comments of requests, I'm assuming that a collection name and description should be enough, but maybe we might want to add an icon/image to avoid couplings with the web application?

**Comment by pcm32** [comment]
Should we have also descriptions for the experiments in the experiments 2 collections table? Bulk Atlas seems to use short descriptions for individual experiments inside the collection.

**Comment by pcm32** [comment]
We could also make the collection identifier a string, so that you can use it for the ?experimentSet=BLUEPRINT sort of use.

File: flyway/scxa/migrations/V8__scxa-create-collections.sql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  		primary key (coll_id);
+
+comment on table collections is 'SCXA Experiment collections';
+
+create unique index experiment_accession_uindex
+	on experiment (accession);
+
+alter table experiment
+	add constraint experiment_pk
+		primary key (accession);
```
Comment (alfonsomunozpomer) [inline flyway/scxa/migrations/V8__scxa-create-collections.sql:22]: Quite an oversight that `experiment` hasn’t got an index!

Comment (pcm32) [reply flyway/scxa/migrations/V8__scxa-create-collections.sql:22]: yeah, I know 😂 !!

**Pull request merged**